test/test-util.js

```diff
@@ -170,12 +170,12 @@ exports.testMatterCustomZap =
 
 exports.totalClusterCount = 111
 exports.totalDomainCount = 20
-exports.totalCommandArgsCount = 1786
```

Comment:
I am always a little cautious about tests changing...could you explain to me why you did this please?

Reply:
Some of the unique constraints in the schema were not being hit because for standard clusters, attributes, etc we had manufacturer code set to NULL and NULL does not equal to NULL in SQL. When I added a derived column to circumvent this, I found a duplicate command in `zcl-builtin/silabs/demo.xml`. With my changes this duplicate command wouldn't be loaded in and hence the command count decreased by one.

Reply:
cool, approved

Reply:
assuming CI passes of course